pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -705,10 +705,20 @@ func createBackupScheduleHook(
 	if !ok {
 		return nil, nil, nil, false, nil
 	}
+
 	eval, err := makeScheduledBackupEval(ctx, p, schedule)
 	if err != nil {
 		return nil, nil, nil, false, err
 	}
+	destinations, errDest := eval.destination()
```

Comment:
We usually name the error variable a generic `err` rather than something that provides context about what the method is doing. Unless ofcourse there is some subtle shadowing going on.